scripts/ingest_regress_test_result-new-format.py

```diff
@@ -32,14 +32,15 @@
     flaky        BOOLEAN NOT NULL,
     arch         arch DEFAULT 'X64',
     lfc          BOOLEAN DEFAULT false NOT NULL,
+    sanitizers   BOOLEAN DEFAULT false NOT NULL,
```

Comment:
I've added `sanitizers` column to the db (it's `false` by default), could you please add setting it from test results? 
You can use LFC as an example, you need to add changes to `scripts/ingest_regress_test_result-new-format.py` and `test_runner/fixtures/parametrize.py` and maybe to workflow to set this env variable for test runs.

Reply:
Please look at a commit I've added to address this: https://github.com/neondatabase/neon/pull/10401/commits/0ac2f2ac34a81a4e9872edfa1a1c6cd8b2318cc1